views/templates/admin/configuration.tpl

```diff
@@ -2,7 +2,7 @@
 </div>
 {if isset($smarty.get.validation)}
 	<div class="conf confirm" style="width: 710px; margin: 0 auto;">
-		{l s='Your Sitemaps were successfully created. Please do not forget to setup the URL' mod='gsitemap'} <a href="{$gsitemap_store_url|escape:'htmlall':'UTF-8'}{$shop->id|intval}_index_sitemap.xml" target="_blank"><span style="text-decoration: underline;">{$gsitemap_store_url|escape:'htmlall':'UTF-8'}{$shop->id|intval}_index_sitemap.xml</a></span> {l s='in your Google Webmaster account.' mod='gsitemap'}
+		{l s='Your Sitemaps were successfully created. Please do not forget to setup the URL' d='Modules.Gsitemap.Admin'} <a href="{$gsitemap_store_url|escape:'htmlall':'UTF-8'}{$shop->id|intval}_index_sitemap.xml" target="_blank"><span style="text-decoration: underline;">{$gsitemap_store_url|escape:'htmlall':'UTF-8'}{$shop->id|intval}_index_sitemap.xml</a></span> {l s='in your Google Webmaster account.' d='Modules.Gsitemap.Admin'}

```

Comment:
Could we use a variable so that the string isn't split in 2?

"Your Sitemaps were successfully created. Please do not forget to setup the URL %URL% in your Google Webmaster account."